faraday/uart/layer_2_protocol.py

```diff
@@ -15,14 +15,20 @@
 import time
 import struct
 import logging
+import sys
 
 # Get configured logger
 logger = logging.getLogger('UARTStack')
 
 
 class layer_2_object(object):
```

Comment:
@reillyeon it appears @kb1lqd used this in [layer 2 service](https://github.com/kb1lqc/Faraday-Software/blob/fec6feae17b3e843459d6582d7040354bba2c51a/faraday/uart/layer_2_service.py#L26) I think it probably can be removed if this line is updated. Am I correct Brent?